pkg/chaosctl/common/common.go

```diff
@@ -309,7 +312,8 @@ func GetPidFromPod(ctx context.Context, pod v1.Pod, daemon v1.Pod) (uint32, erro
 		pfCancel()
 	}()
 
-	daemonClient, err := daemonClient.NewChaosDaemonClientLocally(int(localPort))
+	// TODO: support specify the cert file or get cert file automatically
+	daemonClient, err := NewChaosDaemonClientLocally(int(localPort), "", "", "")
```

Comment:
@Yiyiyimu `chaosctl` will fail to connect to the chaos-daemon after this PR :crying_cat_face: 

Do you have any suggestions on how to specify the `ca`, `cert` and `key`? Should we add command line argument or use environment variable or get them directly from secrets?

Reply:
I'm not so sure that I couldn't say if anyone is better than the other two. 
It seems a bit redundant to use extra three arguments each time we use `chaosctl`. But if we do the settings once with arguments say `chaosctl config --ca=xx`, and then it would be the same with environment variables. Besides, I 'm not sure if directly get secrets from `controller-manager` would break the security.

Have you ever used any other command line tools which support tls? I guess maybe some references could be of help.

Reply:
`viper` seems cool! I'll take a look at it~

Reply:
I suggest that we could implement this feature first. Then migration flag/configuration resolving to `viper`.